ghost/core/core/server/data/migrations/versions/5.84/2024-06-05-08-42-34-populate-post-revisions-custom-excerpt.js

```diff
@@ -0,0 +1,19 @@
+const logging = require('@tryghost/logging');
+const {createTransactionalMigration} = require('../../utils');
+
+module.exports = createTransactionalMigration(
+    async function up(knex) {
+        logging.info('Populating post_revisions.custom_excerpt with post.excerpt');
+
+        await knex.raw(`
+            UPDATE post_revisions
+            JOIN posts ON post_revisions.post_id = posts.id
+            SET post_revisions.custom_excerpt = posts.custom_excerpt
```

Comment:
Not sure if we are too bothered about efficiency here but I think adding a check on `posts.custom_excerpt` (not null / empty) means we could skip updating when theres nothing to update? (`WHERE posts.excerpt !== '' AND posts.excerpt IS NOT NULL`)